lib/engine/game/base.rb

```diff
@@ -1794,6 +1794,12 @@ def add_extra_tile(tile)
         @cities.concat(extra_cities)
         extra_cities.each { |c| @_cities[c.id] = c }
 
+        if (opp_name = tile.opposite&.name) && !new_tile.opposite
+          opp_tile = tile_by_id("#{opp_name}-#{new_tile.index}") || add_extra_tile(tile_by_id("#{opp_name}-0"))
+          opp_tile.opposite = new_tile
+          new_tile.opposite = opp_tile
+        end
+
```

Comment:
Isn't this logic already in update_tile_lists! ?

Reply:
Yes it is, it doesn't need to be here, but this comment lead to me finding a problem with how I implemented it there. I'll close this and open a new PR.